### PR TITLE
remove mpi4py, MPI executor and remote options

### PR DESCRIPTION
**Pull request by azazellochg**
MPI option is removed from STEPS_PARALLEL protocols

### CODE REVIEW

File: pyworkflow/protocol/launch.py [hunk 1]
```diff
@@ -316,22 +229,3 @@ def _run(command, wait, stdin=None, stdout=None, stderr=None):
...
 
     return jobId
-
-
-# ******************************************************************
-# *                 Function related to STOP
-# ******************************************************************
-
-
-def _stopLocal(protocol):
```
Comment (delarosatrevin) [inline pyworkflow/protocol/launch.py:326]: @azazellochg I think this is for stopping protocols when launched to a Queue, not related to MPI

File: pyworkflow/protocol/constants.py [hunk 2]
```diff
@@ -65,7 +65,7 @@
 
 # Steps execution mode
 STEPS_SERIAL = 0      # Execute steps serially, some of the steps can be mpi programs
-STEPS_PARALLEL = 1    # Execute steps in parallel, through threads or mpi
```
Comment (delarosatrevin) [inline pyworkflow/protocol/constants.py:68]: We might consider keeping this Constant for compatibility and avoiding errors in plugins if it is used. We can add a comment about its deprecation

Comment (azazellochg) [reply pyworkflow/protocol/constants.py:68]: the constant is still there, I did not remove it

File: pyworkflow/protocol/launch.py (hunk 1 above)
Comment (azazellochg) [reply pyworkflow/protocol/launch.py:326]: _stopLocal() has benn movied inside stop()

**Comment by pconesa** [comment]
Nice, will need some time to review this but I like it!!

**Review by pconesa — APPROVED** [review]
So far I've been using this branch and it has worked fine. I'm merging it and will keep an eye on buildbot to see if there is any unexpected error due to this chage.

**Pull request merged**